src/dict.c

```diff
@@ -58,7 +58,7 @@
  * between the number of elements and the buckets > dict_force_resize_ratio. */
 static dictResizeEnable dict_can_resize = DICT_RESIZE_ENABLE;
 static unsigned int dict_force_resize_ratio = 5;
-static unsigned int dict_force_shrink_ratio = 10;
+static unsigned int dict_force_shrink_ratio = 2;
```

Comment:
what i meant when i suggested to use 2% was that we can keep using dict_force_resize_ratio and HASHTABLE_MIN_FILL together (i.e `HASHTABLE_MIN_FILL / dict_force_resize_ratio`).
i.e. normally the dict ranges from 10% to 100%, but in extreme cases it'sfrom 2% to 500%, without using any additional constants.

Reply:
Thanks for your patience. I have fixed it.